README.md

```diff
@@ -54,6 +54,24 @@ Example:
 public_path: my/custom/path/
 ```
 
+### Defining public constants through sigil
+You may make individual files public withhin a private package by usage of a comment within the `.rb` file containing `packwerk#publicize!`.
+
+Example:
+
+```ruby
+module Foo
+  class Update
+    # packwerk#publicize!
```

Comment:
I have two preferences for this annotation:
1. That it lives at the top of the file, or at least within the first 5 lines of the file
2. That it looks a bit more like sorbet (which has `typed: true`, for example). Maybe `public_api: true`?

That being said, this is one of the more meaningful, harder to change design decisions, so I wanted to get input from other folks.

cc @shageman , @technicalpickles, @ngan 

I know you folks (especially Ngan) are interested in allowing things to be public via magic comments so this user is graciously helping implement it. I wanted your input on the format of that magic comment.

There are a couple of other places that will require changes related to this sigil. Namely, everything that is coupled to the public folder implementation of privacy.

In the `rubyatscale` org:
- `pack_stats`, example https://github.com/rubyatscale/pack_stats/blob/main/lib/pack_stats/private/metrics/public_usage.rb. (IMO though we can just remove this metric – it has never been useful)
- Other places that mention `public_path` or `app/public`.
  - Org wide search for `app/public` [link](https://github.com/search?q=org%3Arubyatscale%20app%2Fpublic&type=code)
  - Org wide search for `public_path` [link](https://github.com/search?q=org%3Arubyatscale+public_path&type=code)
  - `packs` (the Rust port of packwerk – I could take this one over unless someone is interested in implementing whatever we come up with there

In gusto:
- The published yard docs of public APIs (could be deprecated if a quick poll suggested it's not used)
- ?? maybe other places, but its pretty greppable

Reply:
Thanks @alexevanczuk . The format of the annotation makes sense. I'll make that update and push it back up.